spec/spec_helper.rb

```diff
@@ -4,18 +4,19 @@
 
 require (Pathname.new(__FILE__).dirname + "../lib/suspenders").expand_path
 
-Dir["./spec/support/**/*.rb"].sort.each { |file| require file }
+Dir["./spec/support/*.rb"].sort.each { |file| require file }
 
 RSpec.configure do |config|
-  config.include SuspendersTestHelpers
-  config.include ProjectFiles
+  config.include FeatureTestHelpers, type: :feature
+  config.include GeneratorTestHelpers, type: :generator
 
   config.before(:all) do
-    add_fakes_to_path
-    create_tmp_directory
+    FileOperations.create_tmp_path
```

Comment:
Since we are now including helper modules selectively depending on the "type" metadata, it is more reliable to call helper methods with the fully qualified name at this point.